Oneirocom/Magick
# adding tooltip on hover of labels in the inspector, show tooltip description of the field

## What Changed:

- I added a tooltip on the labels of the inspector
- Show the tooltip short description of the field

## How to test:

- Create a spell and navigate to the inspector view and try to hover on each to see the short description
- Change or click on  a node from the spell view and check on the inspector too while you are hovering.

## Additional information:

[Screencast from 11.07.2023 20:22:08.webm](https://github.com/Oneirocom/Magick/assets/55635977/a0ff43ac-d37c-4d4a-a7c6-b4d8391ccdf3)
